images-list

```diff
@@ -208,6 +209,7 @@ prom/node-exporter rancher/mirrored-prom-node-exporter v1.0.1
 prom/node-exporter rancher/prom-node-exporter v1.0.1
 prom/prometheus rancher/prom-prometheus v2.18.2
 prom/prometheus rancher/mirrored-prom-prometheus v2.12.0
+prom/prometheus rancher/mirrored-prom-prometheus v2.18.2
```

Comment:
This exists in lower section as it moved. @aiyengar2 this is a problem we will run into since we didn't change the image name.

Reply:
Good catch!  
I removed the duplicated line.